### ISSUE TITLE
bootstrap version used in prometheus

### ISSUE THREAD

**Issue by r-madhu** [prometheus/prometheus#5493]
## Proposal
**Use case. Why is this important?**
To fix vulnerability related to bootstrap
*“Nice to have” is not a good use case. :)*

## Bug Report
**What did you do?**
Check vulnerabilities in the iamge
**What did you expect to see?**
Zero High & critical vulnerabilities 
**What did you see instead? Under which circumstances?**
Boot strap version found was 3.3.1. Recommended solution proposed by tool is to upgrade bootstap to 4.3.1 but latest version of prometheus uses 4.1.3 version
**Environment**
Dev
* System information:

	insert output of `uname -srm` here

* Prometheus version:
2.3.2
	insert output of `prometheus --version` here

* Alertmanager version:

	insert output of `alertmanager --version` here (if relevant to the issue)

* Prometheus configuration file:
```
insert configuration here
```

* Alertmanager configuration file:
```
insert configuration here (if relevant to the issue)
```

* Logs:
```
insert Prometheus and Alertmanager logs relevant to the issue here
```

**Comment by jritchieBAE** [comment]
@r-madhu Prometheus is currently using 4.1.3, this was merged in on Feb 20th - it was the latest version at that time - and has been in the release versions since 2.7.2. You haven't said what tool you have used to check for vulnerabilities but you have stated that you are testing against 2.3.2 (which had Bootstrap 3). Do you get similar warnings for vulnerabilities recommending upgrading from 4.1.3 to 4.3.1?

**Comment by simonpasquier** [comment]
If you're referring to [CVE-2019-8331](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-8331), the Boostrap [announcement](https://blog.getbootstrap.com/2019/02/13/bootstrap-4-3-1-and-3-4-1/) says: 

> the data-template attribute for our tooltip and popover plugins lacked proper XSS sanitization of the HTML that can be passed into the attribute’s value.

AFAICT we don't use the `data-template` attribute anywhere so we should be safe.

**Comment by simonpasquier** [comment]
Closed by #5506
